doc/source/flexibility/slowlog_rotation.rst

```diff
@@ -0,0 +1,45 @@
+.. _slowlog_rotation:
+
+========================================
+ Slow Query Log Rotation and Expiration
+========================================
+
+.. note:: 
+
+   This feature is currently considered in technical preview.
+
+This feature was implemented in *Percona Server for MySQL* 8.0.27.
+
+Percona has implemented two new variables, :variable:`max_slowlog_size` and :variable:`max_slowlog_files` to provide users with ability to control the slow query log disk usage. These variables have the same behavior as upstream variable `max_binlog_size <https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_max_binlog_size>`_ and :variable:`max_binlog_files` variable used for controlling the binary log.
+
+.. warning::
+
+   For this feature to work variable :variable:`slow_query_log_file` needs to be set up manually and without the ``.log`` suffix. The slow query log files will be named using :variable:`slow_query_log_file` as a stem, to which a dot and a sequence number will be appended.
```

Comment:
This warning is outdated, it can be removed now.

Reply:
Done.